cmd/osbuild-tests/main_test.go

```diff
@@ -73,9 +73,13 @@ func TestComposeCommands(t *testing.T) {
 	defer os.Remove(uuid.String() + "-logs.tar")
 
 	runComposer(t, "compose", "image", uuid.String())
-	_, err = os.Stat(uuid.String() + "-image.vhdx")
-	require.NoError(t, err, "'%s-image.vhdx' not found", uuid.String())
-	defer os.Remove(uuid.String() + "-image.vhdx")
+	_, err = os.Stat(uuid.String() + "-image.qcow2")
+	require.NoError(t, err, "'%s-image.qcow2' not found", uuid.String())
+	defer os.Remove(uuid.String() + "-image.qcow2")
```

Comment:
Oops I missed this one: `osbuild-tests` is failing because the qcow2 name is `-disk.qcow2`.

Reply:
Thanks!